<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyController</class>
 <widget class="QWidget" name="MyController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>10</y>
     <width>343</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>170</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string> Параметры фрактала                 Константа с </string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>хmin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xminEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xmaxEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>77</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>c</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>xmax</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="maxIterEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>20</width>
         <height>161</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="cEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>77</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>82</x>
         <y>130</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max_iter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="aEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>28</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>103</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>d</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>29</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>a</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>103</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>52</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>53</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="zminEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>zmin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>zmax</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="zmaxEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="yminEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ymin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ymax</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ymaxEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Параметры отрисовщика</string>
      </property>
      <widget class="QLabel" name="visorlabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>53</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет наблюдателя</string>
       </property>
      </widget>
      <widget class="QLabel" name="fonlabel">
       <property name="geometry">
        <rect>
         <x>199</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>54</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
      <widget class="QPushButton" name="foncolorButton">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="visorcolorButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор вида функции</string>
      </property>
      <widget class="QComboBox" name="functionComboBox">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>30</y>
         <width>331</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>q^2 + c</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="drawButton">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Нарисовать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
